### PR TITLE
Add API to get manifest run status

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

**Comment by armenzg** [comment]
👋🏻

File: tests/conftest.py [hunk 1]
```diff
@@ -963,6 +963,64 @@ def test_run_data(bug_data):
...
+    )
+    jt.append(
+        th_models.JobType.objects.create(name="test-windows10-64-2004-qr/opt-mochitest-plain-2")
+    )
+    jt.append(
+        th_models.JobType.objects.create(name="test-windows10-64-2004-qr/opt-mochitest-plain-swr-1")
+    )
+
+    g1 = th_models.Group.objects.create(name="/test")
+    for i in [0, 1, 2]:
```
Comment (Archaeopteryx) [inline tests/conftest.py:983]: `range(3)`, please.

File: tests/conftest.py [hunk 2]
```diff
@@ -963,6 +963,64 @@ def test_run_data(bug_data):
...
+            {
+                'taskcluster_task_id': 'V3SVuxO8TFy37En_6HcXL%s' % i,
+                'taskcluster_retry_id': '0',
+                'name': jt[i].name,
+            }
+        )
+        j = create_jobs([job])[0]
+
+        # when creating the job, we also create the joblog, we want the last job log entry
+        jls = th_models.JobLog.objects.all()
```
Comment (Archaeopteryx) [inline tests/conftest.py:995]: This and the next line can be combined, the `last()` and `latest()` methods are [available](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#last).

**Review by Archaeopteryx — APPROVED** [review]

File: treeherder/webapp/api/groups.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            if item['job_type_name'] not in summary[item['group_name']]:
+                summary[item['group_name']][item['job_type_name']] = {}
+            if item['job_result'] not in summary[item['group_name']][item['job_type_name']]:
+                summary[item['group_name']][item['job_type_name']][item['job_result']] = 0
+            summary[item['group_name']][item['job_type_name']][item['job_result']] += item[
+                'job_count'
+            ]
+
+        data = []
+        mdata = []
```
[the remark this answers is not in the document]
Comment (jmaher) [reply treeherder/webapp/api/groups.py:81]: yes, good catch.

**Pull request merged**